src/github/branch.ts

```diff
@@ -1,33 +1,40 @@
 import { Project } from '../models';
 import transformBranch from '../transforms/branch';
-import parseSmartCommit from '../transforms/smart-commit';
 import reduceProjectKeys from '../jira/util/reduce-project-keys';
-import {Context} from 'probot/lib/context';
+import { Context } from 'probot/lib/context';
+import issueKeyParser from 'jira-issue-key-parser';
+import { isEmpty } from '../jira/util/isEmpty';
 
-export const createBranch = async (context:Context, jiraClient) => {
+export const createBranch = async (context: Context, jiraClient) => {
   const { data: jiraPayload } = await transformBranch(context);
 
   if (!jiraPayload) {
-    context.log({ noop: 'no_jira_payload_create_branch' }, 'Halting futher execution for createBranch since jiraPayload is empty');
+    context.log(
```

Comment:
Not specific to this PR, but should we replace this `context.log` with a `bunyan` logger at some point? Or is it providing more context to the logs?

Reply:
I don't believe it's creating more context but should probably double check. Will do this in a separate PR.